hooks/post-command

```diff
@@ -31,6 +33,17 @@ main() {
     exit 0
   fi
 
+  curl -s https://codecov.io/bash > codecov;
```

Comment:
this would still need the same resiliency as the original fetch: `-s -S --connect-timeout 10 --retry 3 --retry-delay 10` (see changed code) in order to scale on CI.

Reply:
Good idea 👍